scm-ui/ui-webapp/src/repos/sources/containers/Sources.tsx

```diff
@@ -184,6 +193,7 @@ const Sources: FC<Props> = ({ repository, branches, selectedBranch, baseUrl }) =
           branches={branches}
           onSelectBranch={onSelectBranch}
           switchViewLink={evaluateSwitchViewLink()}
+          actions={branches ? renderCodeActions() : null}
```

Comment:
We should use only one place where we put the logic whether the component is rendered or not. Either in the expression or in the function, but not both.